src/lib/input/input.md

```diff
@@ -14,7 +14,7 @@ elements inside `md-form-field` as well. This includes Angular directives such a
 
 The only limitations are that the `type` attribute can only be one of the values supported by
 `mdInput` and the native element cannot specify a `placeholder` attribute if the `md-form-field`
-also contains a `md-placeholder` element.
+also contains an `md-placeholder` element.
```

Comment:
Arguably haha. I'm basing it on the pronunciation of "md" sounding like "em dee". See [APA blog](http://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html) referencing the subject.